thanos/templates/store-deployment.yaml

```diff
@@ -117,7 +144,7 @@ spec:
       volumes:
       - name: data
       {{- if $root.Values.store.dataVolume.backend }}
-        {{- if $root.store.persistentVolumeClaim }}
+        {{- if $root.Values.store.persistentVolumeClaim }}
```

Comment:
This part was broken, found it while testing my changes, so addressed the issue.